src/sssom_schema/schema/sssom_schema.yaml

```diff
@@ -208,7 +207,7 @@ slots:
       - value: skos:broadMatch
         description: "From the SKOS primer: A triple skos:broader (and skos:broadMatch) asserts that , the object of the triple, is a broader concept than , the subject of the triple."
       - value: oboInOwl:hasDbXref
-        description: Two terms are related in some way. The meaning is frequently consistent across a single set of mappings. Note this property is often overloaded even where the terms are of a different nature (e.g. interpro2go)
```

Comment:
shouldn't lose the trailing close parentheses

Reply:
Thx.